provisioning/roles/castlemind/tasks/newrelic.yml

```diff
@@ -0,0 +1,6 @@
+- name: ensure newrelic_agent.log exists
+  file: path={{ app_root }}/{{ nodeapp_name }}/newrelic_agent.log state=touch owner={{ nodeapp_user }} mode=0644
+
+- name: restart node app
+  service: name={{ nodeapp_name }} state=restarted
```

Comment:
This is brute force—it would be much more graceful to restart using a [handler](http://docs.ansible.com/playbooks_intro.html#handlers-running-operations-on-change)